packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -158,48 +161,99 @@ export default class OverTheBoxDetailsCtrl {
     };
   }
 
+  static getMaxSpeed(series) {
+    let max = 0;
+    let currentMax = 0;
+    let rateUnit = 'Mbps';
+    let rate = 0;
+    if (isArray(series) && series.length && series[0].points) {
+      series[0].points.forEach((point) => {
+        currentMax = 0;
+        for (let i = 0; i < series.length; i += 1) {
+          for (let j = 0; j < series[i].points.length; j += 1) {
+            if (series[i].points[j].timestamp === point.timestamp) {
+              currentMax += series[i].points[j].value;
+            }
+          }
+        }
+        max = max > currentMax ? max : currentMax;
+      });
+      rate =
+        Math.round(currentMax / this.OVERTHEBOX_DETAILS.convertToMbps) / 10;
+      if (!rate) {
+        rate =
+          Math.round(currentMax / this.OVERTHEBOX_DETAILS.convertToKbps) / 10;
```

Comment:
Shouldn't be `max` used here?
```suggestion
        Math.round(max / this.OVERTHEBOX_DETAILS.convertToMbps) / 10;
      if (!rate) {
        rate =
          Math.round(max / this.OVERTHEBOX_DETAILS.convertToKbps) / 10;
```

Reply:
No, it's not `max` be used here.